dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
```

Comment:
Dependency and Security: Dependabot can handle generating PRs to address security risks tied to a dependency, as well as dependency modernization for when viable updates are available. They do not leverage the same configuration, they are separate. Both need to be included.

Reply:
[Configuring Dependabot security updates - GitHub Docs](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates)

According to this, security updates can be enabled without a dependabot.yml file in settings.  That would just need to be done in the organization level, but each repo can have its own settings as well.

Reply:
So, general configuration can be handled that way it looks like, I agree. But down at the very bottom of that page it says you can drive security updates configuration via the dependabot file still, but that your configuration distinction is the provided directory and no target-branch.